CMakeLists.txt

```diff
@@ -22,15 +24,33 @@ set(tinyobjloader-examples-objsticher
 	${TINYOBJLOADEREXAMPLES_DIR}/obj_sticher/obj_sticher.cc
 	)
 
+#Install destinations
+set(TINYOBJLOADER_CMAKE_DIR lib/cmake)
```

Comment:
I can do, but it is up to @syoyo whether they would like to change the install location.